tutor/src/models/user/menu.js

```diff
@@ -9,6 +9,7 @@ const ROUTES = {
 
   myCourses: {
     label: 'My Courses',
+    locked(course) { return get(course, 'currentRole.isTeacherStudent'); },
```

Comment:
My one comment about this (more for UX really) is that they can escape anyway if they bookmarked the `My Courses` page.

Reply:
yeah, there's multiple ways they can escape, but I think that's really ok.  With the frame it's super clear that they're in a student view